docker-compose.yml

```diff
@@ -0,0 +1,72 @@
+version: '3.6'
+
+services:
+  frontend:
+    container_name: joladnijo_frontend_${ENVIRONMENT}
+    restart: always
+    expose:
+      - 80
```

Comment:
Ha jol sejtem, akkor meg lesz egy nginx service is a teljes stackben, es az fogja proxyzni a frontend service fele a forgalmat. Viszont az expose-nak van meg ertelme? Ugyanazon a networkon ugy tudom, hogy expose nelkul is elerhetoek lesznek a containerek.